content/programming-cookbook/vim-and-neovim/vim-split-windows.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Vim split windows Horizontally and Vertically
+showMetadata: true
+editable: true
+showToc: true
+---
+
+## You need to split the current windows
+
+### Split Horizontally
+
+you can use `<Ctrl-w> + s` or `:sp` colon and type sp to split the window horizontally
```

Comment:
Please update all fragments to proper English sentences by starting with a capital letter and ending with period.
Thanks.